internal/bootstrap/bootstrap_test.go

```
@@ -0,0 +1,351 @@
+	for _, kind := range []Kind{KindAgentControl, KindAgentContracts} {
+		t.Run(string(kind), func(t *testing.T) {
+			params.Repository = string(kind)
+			rendered, err := Render(kind, params)
+			if err != nil {
+				t.Fatalf("Render() error = %v", err)
+			}
+			root := t.TempDir()
+			if err := Write(root, rendered); err != nil {
+				t.Fatalf("Write() error = %v", err)
+			}
+			command := exec.Command(python, "tools/validate.py")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)